Project.toml

```diff
@@ -11,7 +11,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-ChainRulesCore = "0.8"
+ChainRulesCore = "0.9.1"
```

Comment:
Does this bump mean this PR is blocked by #44?

Reply:
Hmmm possibly. There's no particular hurry with this though, so I'm happy to wait until it's merged.